src/message/fetchActions.js

```diff
@@ -251,6 +263,7 @@ const fetchPrivateMessages = () => async (dispatch: Dispatch, getState: GetState
       numAfter: 0,
       foundNewest: found_newest,
       foundOldest: found_oldest,
+      ownUserId: getOwnUserId(getState()),
```

Comment:
> Still, I don't think it's a very bad thing to throw here; the alternative is allowing race condition where the result of this message fetch is applied when the account it's meant for is no longer logged in.

Er, well, of course it's bad in that it would cause a crash—but having corrupted data is also quite bad. 🙂

Reply:
Hmm true! Yeah, we should really fix this possible case in general by cancelling these requests (#4170), but we haven't yet.

I'll note this in the commit message.